Sources/PT.PM.Dsl/DslUstConverter.cs

```diff
@@ -175,10 +175,10 @@ public PatternUst VisitMethodDeclaration(DslParser.MethodDeclarationContext cont
             PatternUst result;
             IEnumerable<PatternUst> modifiers = ProcessLiteralsOrPatternIds(context._modifiers);
             PatternUst name = VisitLiteralOrPatternId(context.methodName);
-            var arbitraryDepthExpression = context.arbitraryDepthExpression();
-            if (arbitraryDepthExpression != null)
+            var bodyExpression = context.expression();
+            if (bodyExpression != null)
             {
-                var body = (PatternArbitraryDepth)VisitArbitraryDepthExpression(arbitraryDepthExpression);
+                var body = VisitExpression(bodyExpression);
```

Comment:
Use concrete type instead of `var`. The type is not obvious from the right part (method invocation).

Reply:
There can be any expression

Reply:
`PatternUst body = VisitExpression(bodyExpression);`